scripts/chk_shellscripts/chk_shellscripts.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -e
+
+REPO_ROOT="$(dirname "$0")"/../..
+REPO_ROOT=$(realpath "${REPO_ROOT}")
+IGNORE_FILENAME="ignore.txt"
+IGNORE_FILE="${REPO_ROOT}/scripts/chk_shellscripts/${IGNORE_FILENAME}"
+
+IGNORE=$(cat "${IGNORE_FILE}")
+
+cd "${REPO_ROOT}"
+# shellcheck disable=SC2044
+for FILE in $(find . -type f -name "*.sh")
```

Comment:
Can we just use `join` to filter out the files to be ignored?
https://unix.stackexchange.com/a/120428

Reply:
cool! didn't knew `join`